src/peft/peft_model.py

```diff
@@ -149,7 +156,9 @@ def save_pretrained(self, save_directory, **kwargs):
             peft_config.inference_mode = inference_mode
 
     @classmethod
-    def from_pretrained(cls, model, model_id, adapter_name="default", is_trainable=False, **kwargs):
+    def from_pretrained(
+        cls, model, model_id, adapter_name="default", is_trainable=False, load_safetensors=False, **kwargs
```

Comment:
```suggestion
        cls, model, model_id, adapter_name="default", is_trainable=False, use_safetensors=False, **kwargs
```

For consistency with `transformers` and `diffusers`. And it should probably be `None` instead of `False` if we want to make it the default.